server/src/main/java/org/elasticsearch/telemetry/Trace.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.telemetry;
+
+import org.elasticsearch.common.UUIDs;
+import org.elasticsearch.core.Releasable;
+import org.elasticsearch.telemetry.tracing.Traceable;
+import org.elasticsearch.telemetry.tracing.Tracer;
+import org.elasticsearch.threadpool.ThreadPool;
```

Comment:
so I am planning to move the telemetry package into its own `:lib` this import will make it impossible.